.github/ISSUE_TEMPLATE/release.yml

```diff
@@ -0,0 +1,57 @@
+name: Release
+description: Propose a new minor/major release
+title: "[Release]: "
+labels: ["release"] #Verified - label exists
+body:
+  - type: markdown
+    attributes: 
+      value: |
+        This issue is to help the committers to the Jakarta Concurrency project to propose
+        and complete the release process.
+  - type: textarea
+    attributes:
+      label: Proposal
+      description: |
+        Why is a release needed?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Outstanding work
+      description: |
+        Issues / pulls to finish before this release can become generally available.
+      value: |
+        - [ ] 
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Check List
+      description: |
+        This is a checklist of what needs to be completed during the release process.
+        This checklist typically does not need to be edited prior to submitting this form.
+      value: |
+          #### Prepare
+            - [X] Open this issue.
+            - [ ] Complete outstanding work
+          #### Stage release
+            - [ ] [Build and stage artifacts to staging repository](https://ci.eclipse.org/cu/view/Release/job/concurrency_api_1-build-and-stage/)
+            - [ ] [Build and stage the TCK distribution artifact to download.eclipse.org](https://ci.eclipse.org/cu/view/Release/job/concurrency_tck_1-build-and-stage/)
+            - [ ] Update this issue with a comment that links to the staged artifacts and the generated GitHub branch and tag
+            - [ ] [Create a `pre-release` on GitHub](https://github.com/jakartaee/concurrency/releases/new)
+          #### Verify and modify
+            - [ ] Ask for feedback from the community to verify the staged artifact has all the expected changes.
+            - [ ] If anything needs to be added before publishing do that now and repeat the `Stage Release` section
+          #### Publish release
+            - [ ] [Publish staged artifacts to the public repository](https://ci.eclipse.org/cu/view/Release/job/concurrency_api_3-staging-to-release/)
+            - [ ] [Publish staged TCK distribution artifact to download.eclipse.org](https://ci.eclipse.org/cu/view/Release/job/concurrency_tck_2-staging-to-promoted/)
+            - [ ] Update this issue with a comment that links to the published artifacts and the generated GitHub branch and tag
+            - [ ] [Modify the `pre-release` on GitHub to be `latest`](https://github.com/jakartaee/concurrency/releases)
+          #### Follow up
+            - [ ] [Email a Jakarta steering committee member to promote the TCK Distribution publically](https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/)
```

Comment:
Any ideas for how someone who doesn't know a steering committee member could do this?  I checked for a mailing list or some other way of contacting the steering committee, but didn't find anything useful. Maybe add something like "tag other committers in an issue comment if you need help contacting a committee member"?

Reply:
Yeah I like the recommendation of emailing the CU mailing list